scene/gui/text_edit.cpp

```diff
@@ -472,7 +472,7 @@ String TextEdit::Text::get_custom_word_separators() const {
 }
 
 String TextEdit::Text::get_default_word_separators() const {
-	String concat_separators = "´`~$^=+|<>";
+	String concat_separators = "`~!@#$%^&*()-=+[{]}|;:'\",.<>/?";
```

Comment:
Maybe those can be excluded with sub-ranges of the ASCII table?

BTW looking at the list, aren't we missing `\` (which would need to be escaped as `\\`)?

Reply:
> Maybe those can be excluded with sub-ranges of the ASCII table?

Do you mean iterating through char-codes instead of hardcoding them?

> BTW looking at the list, aren't we missing \ (which would need to be escaped as \\)?

Yes! You are right. Will be added as well. Iterating would have prevented that, probably :).

Reply:
> > Maybe those can be excluded with sub-ranges of the ASCII table?
> 
> Do you mean iterating through char-codes instead of hardcoding them?
> 
> > BTW looking at the list, aren't we missing \ (which would need to be escaped as \)?
> 
> Yes! You are right. Will be added as well. Iterating would have prevented that, probably :).

Yes, though looking at the table it's not a single block so it might not be worth it.
Maybe we can at least list them in the same order as in the ASCII table so that it's easy to spot which ones are included or left out (in particular `_` which from a quick look is the only punctuation from the table that we're not including, as we want to support `snake_case` identifiers I suppose).